src/test/resources/conf/local.conf.json

```diff
@@ -4,12 +4,14 @@
   "key": "BROWSERSTACK_ACCESS_KEY",
 
   "capabilities": {
-    "build": "cucumber-java-browserstack",
-    "browserstack.debug": true,
-    "browserstack.local": true
+    "bstack:options" : {
+      "buildName" : "cucumber-java-browserstack",
+      "local" : "true",
+      "debug" : "true"
+    }
```

Comment:
Cosmetic change:  can remove the space between key and `:` to have it the same format throughout the file, and in the other conf file as well.
So like `"local": "true"` in place of `"local" : "true"`